src/librustc_borrowck/diagnostics.rs

```diff
@@ -386,6 +386,109 @@ fn foo(a: &mut i32) {
     let bar = || {
         inside_closure(a)
     };
+```
```

Comment:
this is causing compilation to fail; it makes the error above invalid. you need to include a closing }